### PR TITLE
Added Feedbacks  fixes #21

### LINKED ISSUE #21: Add successful statement after executing each command
While creating file we get this output as : `File successsfully created!!!`
file

However for some command like deleting file` df ` it doesn,t show any output
fileoriginal

Check for all the functions and add the specific execution statement at last.

### PR DESCRIPTION
**Pull request by thesmallstar**
Fixes #21

### CODE REVIEW

File: lib/functions.js [hunk 1]
```diff
@@ -7,13 +7,18 @@ exports.deletefile = async () => {
   const input = await question.questions();
   const { filename } = input;
   const name = filename;
-  shell.rm(name);
+  var x;
+  x = shell.rm(name);
+  if (x.code) console.log("The file was was not deleted. ");
+  else console.log("The file was deleted ");
```
Comment (debck) [inline lib/functions.js]: Use `chalk` inside console.log to display the messages.

File: lib/functions.js [hunk 2]
```diff
@@ -25,6 +30,9 @@ exports.renamefile = async () => {
 
   fs.rename(name, newname, function(err) {
     if (err) console.log("ERROR: " + err);
+    else {
+      console.log("file was renamed.");
```
Comment (debck) [inline lib/functions.js]: Here also.. Use `chalk` inside console.log to display the messages.

File: lib/functions.js [hunk 3]
```diff
@@ -7,13 +7,18 @@ exports.deletefile = async () => {
...
+  else console.log("The file was deleted ");
 };
 
 exports.deletedirec = async () => {
   const dir_res = await ifdir.getdir();
   const { dirname } = dir_res;
-  shell.rm("-rf", dirname);
+  x = shell.rm("-rf", dirname);
+  if (x.code) console.log("The folder was was not deleted. ");
+  else console.log("The folder was deleted ");
```
Comment (debck) [inline lib/functions.js]: Here too .Use `chalk` inside console.log to display the messages.

**Comment by debck** [comment]
resolve the changes requested and also remember to add `Fixes #(issue_number)` whenever you make a PR

File: lib/functions.js (hunk 1 above)
Comment (debck) [reply lib/functions.js]: Like this 
```
console.log(
      chalk.blue("Directory created successfully") + chalk.red("!!!")
    );
```

**Comment by debck** [comment]
When you write `Fixes #<isssue_number>` it automatically closes the issue when the PR is merged 

capture

**Comment by thesmallstar** [comment]
Woah, I didn't know that.  :)

**Pull request merged**